luthier/config/src/metricMakers.lua

```diff
@@ -56,8 +55,8 @@ for _, operation in ipairs({"PLUS","MINUS","MULTIPLY","DIVIDE"}) do
     }
 end
 
-for _, grain in ipairs {"HOUR", "DAY"} do
-    M["aggregateAverage" .. grain] = {
+for _, grain in ipairs {"Hour", "Day"} do
```

Comment:
The enums are constant case 'HOUR' and 'DAY', do we want to use title case here?

Reply:
We should, because it's being used for the metric name and "aggregateAverageHOUR" is a pain in the butt metric name. 

Frankly, I think we should make it a policy of the builtin stuff at least that any time we reference enum values defined in Java, we can do so in a case insensitive manner.